.prettierignore

```diff
@@ -0,0 +1,2 @@
+lib/octane-rules.js
```

Comment:
Adding these files to a .prettierignore as they are autogenerated

Reply:
Isn't adding these files to `.eslintignore` sufficient? Let's avoid creating this new `.prettierignore` file if it's not necessary. If there's some benefit to this new file, let me know.

Reply:
So this is solving the problem I was seeing where if I saved either of these files by mistake in my editor Prettier would autoformat them. The files are auto-generated so they don't follow the repo style guide (they use double instead of single quotes).

.eslintignore doesn't solve this problem it just opts them out of linting.